components/project-card/project-card-detailed.scss

```diff
@@ -0,0 +1,172 @@
+    cursor: pointer;
+
+    &.beating {
+      animation-name: beating;
+      animation-duration: 0.4s;
+      animation-fill-mode: both;
+    }
+  }
+
+  &.bookmarked .circle-heart {
+    background-image: url(/assets/svg/icon-bookmark-selected-entry-detailed.svg);
+  }
+
+  .issues-and-tags-container {
+    @include scaleText(16px, 24px, 20px, 28px);
+
+    color: $dark-blue;
+
+    @media (min-width: $bp-md) {
+      font-weight: 300;
+    }
+  }
+
+  .get-involved {
+    background-color: $gray-05;
+  }
+
+  .description {
+    p {
+      margin-bottom: 0;
+    }
+  }
+
+  .thumbnail-wrapper {
+    grid-area: image;
+
+    @media (min-width: $bp-md) {
+      grid-column: 1 / 2;
+      grid-row: 2;
+    }
+
+    @media (min-width: $bp-lg) {
+      grid-column: 1 / 13;
+      grid-row: 1;
+    }
+  }
+
+  .title-author-wrapper {
+    grid-area: info;
+
+    @media (min-width: $bp-lg) {
+      grid-column: 10 / 25;
+      grid-row: 1;
+      z-index: 1;
+      align-self: center;
+    }
+  }
+
+  .social-panel-wrapper {
+    grid-area: icons;
+    z-index: 1;
+  }
+
+  .thumbnail-title-social {
+    display: grid;
+    grid-template:
+      [row1-start] "image image image" 1fr [row1-end]
+      [row2-start] ". info ." 0.5fr [row2-end]
+      [row3-start] ". icons ." 40px [row3-end]
+      / 15px 1fr 15px;
+
+    @media (min-width: $bp-sm) {
+      grid-template-columns: 100px 1fr 100px;
+    }
+
+    @media (min-width: $bp-md) {
+      margin-top: 48px;
+      grid-template:
+        [row1-start] "info info" 0.5fr [row1-end]
+        [row2-start] "image icons" 1.5fr [row2-end]
+        / 2fr 1fr;
+    }
+
+    @media (min-width: $bp-lg) {
+      grid-template-columns: repeat(24, 1fr);
+      grid-template-rows: 0.25fr 0.25fr;
+      grid-row-gap: 48px;
+    }
+
+    &-wrapper {
+      @media (min-width: $bp-md) {
+        width: 720px;
+        max-width: 100%;
+        padding: 0 15px;
+        margin: 0 auto;
+      }
+
+      @media (min-width: $bp-lg) {
+        width: 960px;
+      }
+
+      @media (min-width: $bp-xl) {
+        width: 1140px;
+      }
+    }
+  }
+
+  .title-author {
+    @media (min-width: $bp-lg) {
+      background-color: $white;
+      padding: 24px 0 24px 24px;
+    }
+  }
+
+  .social-panel {
+    @media (min-width: $bp-lg) {
+      position: sticky;
+      top: 50px;
+    }
+  }
+
+  .action-panel {
+    @media (min-width: $bp-lg) {
+      width: 80%;
+    }
+
+    a {
+      @media (min-width: $bp-lg) {
+        width: 49%;
+      }
+
+      &:first-of-type {
+        width: 100%;
+
+        @media (min-width: $bp-md) {
+          width: auto;
+        }
+
+        @media (min-width: $bp-lg) {
+          width: 49%;
+        }
+      }
+    }
+  }
+
+  .help-needed-info {
+    a {
+      font-weight: 700;
+    }
+  }
+
+  .description,
+  .interest,
+  .help-needed,
+  .issues-and-tags,
+  .time-posted,
+  .report-correction {
+    @media (min-width: $bp-lg) {
+      width: 80%;
+    }
+  }
```

Comment:
Why 80%? If the goal is to balance out these sections in terms of spacing we should be able to achieve that via Bootstrap's class. Don't combine Boostrap's column layout with `%` width as their final absolute values aren't always equivalent.

In `project-card-detailed.jsx` offset for these sections was added via Boostrap's `offset-lg-2` class. So to make this work you should be able to just add `col-lg-8` to the class list.

Reply:
The only goal here was to decrease the width of the main content. I think that this can be done either way, however, if I were to use `col-lg-...`, I'd have to remove the additional padding it adds _that I don't need_. I've used Bootstrap as sparingly as possible and this is a good example of that. `offset-lg-2` adds a `margin-left: 16.66667%` which is all that's needed and was purely used for positioning. So, I'd prefer not to use `col-...` since it adds extra styling that isn't necessary.